python/Instruction_Memory.py

```diff
@@ -2,9 +2,12 @@
 
 @block
 def Instruction_Memory(clk, Addr, Out): #Addr and clk input, out is output
+    
+    mem = [intbv(min = 32, max = 32)] * 256 #create 256 32-bit memory array
 
     @always(clk.posedge)
     def execute():
-        return
+        Out = mem[address[9:2]] #return the address of the instruciton
```

Comment:
Out should be a signal, in which case you would use Out.next = ... for assignment.